csm_web/scheduler/views.py

```diff
@@ -170,13 +170,13 @@ def attendances(self, request, pk=None):
 
 
 class ProfileViewSet(*viewset_with('list')):
-    queryset = None
     serializer_class = None
 
+    def get_queryset(self):
+        pass
+
```

Comment:
For some strange reason DRF complaining with an exception when `queryset = None`, but this used to work. I'm guessing updating DRF caused this new behavior, but this little hack fixes it.

Reply:
wouldn't setting `queryset = EmptyQuerySet()` be more idiomatic?

Reply:
Didn't know `EmptyQuerySet` existed, will change